Reference to MDE submissions (#9070)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/submitting-malware-and-non-malware-to-microsoft-for-analysis.md

ms.prod: m365-security
---

> # Submit malware, non-malware and other susipicious files to Microsoft for analysis

Reviewer: ```suggestion
# Submit malware, non-malware, and other suspicious files to Microsoft for analysis
```

- Avoid opening messages that look suspicious.
- Never open an attachment from someone you don't know.
- Avoid opening attachments in messages that urge you to open or click them.

> - Avoid opening files downloaded from the net unless from a verified source

Reviewer: ```suggestion
- Avoid opening files downloaded from the internet unless they are from a verified source.
```

- Never insert files on an anonymous pen drive.

> But what can you do if you receive a message with a suspicious attachment or got a suspicious file on your system? Or what if you suspect that your computer or device was infected by an email attachment that made it past our filters or a file you downloaded from the internt? In these cases, you should submit the suspicious attachment or file to Microsoft. Conversely, if an attachment in an email message or file was incorrectly identified as malware or some other threat, you can submit that, too.

Reviewer: ```suggestion
But what can you do if you receive a message with a suspicious attachment or have a suspicious file on your system? Or what if you suspect that your computer or device was infected by an email attachment that made it past our filters or a file you downloaded from the internet? In these cases, you should submit the suspicious attachment or file to Microsoft. Conversely, if an attachment in an email message or file was incorrectly identified as malware or some other threat, you can submit that, too.
```

## What do you need to know before you begin?

- Messages with attachments that contain scripts or other malicious executables are considered malware, and you can use the procedures in this article to report them.

- Messages with links to malicious sites are considered spam. For more information about reporting spam and non-spam, see [Report messages and files to Microsoft](report-junk-email-messages-to-microsoft.md).

> - Files that block you from your accessing your system and demand money to open it are considered ransomware. 

Reviewer: ```suggestion
- Files that block you from your accessing your system and demand money to open them are considered ransomware. 
```

## Submit malware files to Microsoft

> For organizations with Microsoft 365 Defender Subscription or Microsoft 365 Defender for Endpoint Plan 2 or Microsoft 365 Defender for Office Plan 2, you can submit files using **Submissions** page in the Microsoft 365 Defender portal. For more information, see [Use admin submission for submitting files in Microsoft Defender for Endpoint](admin-submissions-mde.md).

Reviewer: ```suggestion
Organizations that have a Microsoft 365 Defender subscription, Microsoft 365 Defender for Endpoint Plan 2, or Microsoft 365 Defender for Office Plan 2 can submit files using the **Submissions** page in the Microsoft 365 Defender portal. For more information, see [Use admin submission for submitting files in Microsoft Defender for Endpoint](admin-submissions-mde.md).
```

> Otherwise go to the Microsoft Security Intelligence website at <https://www.microsoft.com/wdsi/filesubmission> to submit the file. To receive analysis updates, sign into the website, or enter a valid email address. We recommend that you use your Microsoft work or school account.

Reviewer: ```suggestion
Alternatively, go to the Microsoft Security Intelligence website at <https://www.microsoft.com/wdsi/filesubmission> to submit the file. To receive analysis updates, sign in to the website, or enter a valid email address. We recommend that you use your Microsoft work or school account.
```

## Submit non-malware files to Microsoft

> For organizations with Microsoft 365 Defender Subscription or Microsoft 365 Defender for Endpoint Plan 2 or Microsoft 365 Defender for Office Plan 2, you can submit files using **Submissions** page in the Microsoft 365 Defender portal. For more information, see [Use admin submission for submitting files in Microsoft Defender for Endpoint](admin-submissions-mde.md).

Reviewer: ```suggestion
Organizations that have a Microsoft 365 Defender Subscription, Microsoft 365 Defender for Endpoint Plan 2, or Microsoft 365 Defender for Office Plan 2 can submit files using the **Submissions** page in the Microsoft 365 Defender portal. For more information, see [Use admin submission for submitting files in Microsoft Defender for Endpoint](admin-submissions-mde.md).
```

> Otherwise go to the Microsoft Security Intelligence website at <https://www.microsoft.com/wdsi/filesubmission> to submit the file. To receive analysis updates, sign into the website, or enter a valid email address. We recommend that you use your Microsoft work or school account.

Reviewer: ```suggestion
Alternatively, go to the Microsoft Security Intelligence website at <https://www.microsoft.com/wdsi/filesubmission> to submit the file. To receive analysis updates, sign in to the website, or enter a valid email address. We recommend that you use your Microsoft work or school account.
```

> You can submit a file that you believe was incorrectly identified as malware to the website (just select **No** for the question, **Do you believe this file contains malware?**).

Reviewer: ```suggestion
You can also submit a file that you believe was incorrectly identified as malware to the website (just select **No** for the question, **Do you believe this file contains malware?**).
```
